### PR TITLE
Misc lemming fixes

### PR DESCRIPTION
**Pull request by DanG100**
* new operator version
* expose more services by default
* test coverage

### CODE REVIEW

File: topo/node/lemming/lemming.go [hunk 1]
```diff
@@ -99,14 +99,15 @@ func (n *Node) Create(ctx context.Context) error {
...
-				SelfSigned: lemmingv1.SelfSignedSpec{
+			dut.Spec.TLS = &lemmingv1.TLSSpec{
+				SelfSigned: &lemmingv1.SelfSignedSpec{
 					CommonName: tls.SelfSigned.CommonName,
 					KeySize:    int(tls.SelfSigned.KeySize),
 				},
 			}
 		}
 	}
+	log.Warningf("lemming spec: %+v", dut)
```
Comment (alexmasi) [inline topo/node/lemming/lemming.go:110]: is this intended?

**Review by alexmasi — COMMENTED** [review]
This will fail the presubmit because it will pull version 1 instead of 2 of the operator, this is known issue with the presubmit so I will ignore it. Did you test this locally to confirm?

File: topo/node/lemming/lemming.go (hunk 1 above)
Comment (DanG100) [reply topo/node/lemming/lemming.go]: nope, debugging leftover removed

**Comment by DanG100** [comment]
> This will fail the presubmit because it will pull version 1 instead of 2 of the operator, this is known issue with the presubmit so I will ignore it. Did you test this locally to confirm?

yup, it's working for me and in our presubmit

**Review by alexmasi — APPROVED** [review]

**Pull request merged**